packages/steiger-plugin-fsd/src/forbidden-imports/index.ts

```diff
@@ -1,12 +1,13 @@
 import * as fs from 'node:fs'
 import { layerSequence, resolveImport } from '@feature-sliced/filesystem'
 import precinct from 'precinct'
-const { paperwork } = precinct
```

Comment:
issue: this was deliberately written like that. It's an unfortunate limitation of esbuild that we have to work around by doing this, and the point is to show that the import is actually called `paperwork`, it's just that esbuild cannot recognize it

suggestion: roll this change back